android/MobiledgeXSDKDemo/computervision/src/main/java/com/mobiledgex/computervision/ObjectClassRenderer.java

```diff
@@ -0,0 +1,197 @@
+/**
+ * Copyright 2019 MobiledgeX, Inc. All rights and licenses reserved.
+ * MobiledgeX, Inc. 156 2nd Street #408, San Francisco, CA 94105
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mobiledgex.computervision;
+
+import android.content.Context;
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.graphics.Rect;
+import android.util.AttributeSet;
+import android.util.Log;
+import android.view.View;
+import android.view.animation.Animation;
+import android.view.animation.AnimationUtils;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Draws a set of rectangular coordinates with class names of detected objects in multiple colors.
+ */
+public class ObjectClassRenderer extends View {
+    private static final String TAG = "ObjectClassRenderer";
+    public static final int DEFAULT_STROKE_WIDTH = 6;
+    private int mStrokeWidth = DEFAULT_STROKE_WIDTH;
+    private int mTextSize = (int) (getResources().getDisplayMetrics().scaledDensity * 16);
+    private JSONArray mObjects;
+    private int mWidth;
+    private int mHeight;
+    private int mWidthOff;
+    private int mHeightOff;
+    private float mServerToDisplayRatioX;
+    private float mServerToDisplayRatioY;
+    private boolean mMirrored;
+    private Animation mAlphaAnim;
+
+    /**
+     * Array for drawing each object with a different color.
+     */
+    private String[] colors = {
```

Comment:
Bit hard coded?

Reply:
I'm open to suggestions for an alternative means of having an array of pleasant colors.

Reply:
These are from https://matplotlib.org/examples/color/colormaps_reference.html, Qualitative, "tab10" set, followed by the 10 lighter variants from "tab20".

Reply:
Your comment prompted me to add a check so we don't try to pull more colors than the array holds.